WooCommerce/Classes/Extensions/UIImage+Woo.swift

```diff
@@ -427,6 +427,12 @@ extension UIImage {
         return UIImage.gridicon(.types).imageFlippedForRightToLeftLayoutDirection()
     }
 
+    /// Visibility Image
+    ///
+    static var visibilityImage: UIImage {
+        return UIImage.gridicon(.visible)
+    }
```

Comment:
This seems a duplicated icon since we use it inside the Product Settings Password row. It's called `passwordFieldImage`. What do you think about removing `passwordFieldImage` and substitute it with `visibilityImage`?

Reply:
good point, updated here: https://github.com/woocommerce/woocommerce-ios/pull/2593/commits/4d3453cdd435b2c49093f804084619d06d24e1a8